pynest/nest/__init__.py

```diff
@@ -91,6 +91,10 @@ def __init__(self, name):
         _rel_import_star(self, ".lib.hl_api_simulation")
         _rel_import_star(self, ".lib.hl_api_spatial")
         _rel_import_star(self, ".lib.hl_api_types")
+        try:
+            _rel_import_star(self, ".server.hl_api_server")
```

Comment:
Sorry for the confusion @babsey but I think the code you're looking for in `__init__.py` is just:

```python
            type(self).server = _lazy_module_property("server")
```

This will make all members in `pynest/nest/server` available as `nest.server.*`, you can then use `nest.server.app` as WSGI application. 

The code you have here

```
_rel_import_star(self, ".server.hl_api_server")
```

is equivalent to `from .server.hl_api_server import *` and I'm not sure that's what supposed to happen here. For example if `nest.server.app` is available as `nest.app` I'm not sure that it's clear anymore that we're dealing with a NEST Server WSGI app.

Reply:
I also tried to use only `type(self).server = _lazy_module_property("server")` but it shows error when I want to start NEST Server:

```
***
*** WARNING: NEST Server runs without a RestrictedPython trusted environment.
***
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 979, in _find_and_load_unlocked
  File "/home/spreizer/opt/nest-server/lib/python3.8/site-packages/nest/__init__.py", line 427, in _setattr_error
    cls_attr.__set__(self, val)
AttributeError: can't set attribute
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 137632, cores: 1)
```

I am currently experimenting alternative solution, but no success so far.

Reply:
Can you print out which parts are involved in that error? Which attribute is being set etc? Do you have anything I can reproduce this with.